### ISSUE TITLE
Devise for Authentication?

### ISSUE THREAD

**Issue by kidbombay** [open-ventures/seed#3]
Can I recommend we use Devise https://github.com/plataformatec/devise for authentication?
It's well known and we would get login/sign up/forgot pass/easy oauth support for FB/twitter login pretty easily.

**Comment by khadijah-coakley** [comment]
Yeah....for on-boarding we'll likely need to bake in something like [Wicked](https://github.com/schneems/wicked). We usually code authentication without device to avoid issues with overriding the user edit and being able to streamline the user creation flow but I'm cool with either.

**Comment by kidbombay** [comment]
Wicked is awesome.
